keps/sig-node/727-resource-metrics-endpoint/kep.yaml

```diff
@@ -2,18 +2,31 @@ title: Kubelet Resource Metrics Endpoint
 kep-number: 727
 authors:
   - "@dashpole"
+  - "@richabanker"
 owning-sig: sig-node
 participating-sigs:
   - sig-instrumentation
+status: implemented
+creation-date: 2019-01-24
+last-updated: 2023-09-08
 reviewers:
   - DirectXMan12
   - tallclair
 approvers:
   - dchen1107
   - brancz
-creation-date: 2019-01-24
-last-updated: 2019-02-21
-status: implementable
+replaces:
+  - none
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: "stable"
```

Comment:
It's not natural to go directly from Alpha to GA. But given the feature is effectively "always on and not possible to disable it", it may be seen as already GA.

So let's actually go with it.

Reply:
+1 and thanks for spotting this. I actually thought it was GA-ed long time back.